templates/schedule.html

```diff
@@ -24,20 +25,17 @@ <h1>Get your tickets now</h1>
     <div id="programme">
         <div class="content-wrapper">
             <div class="content">
-                <h2>tbd</h2>
-        <!--
-        <pretalx-schedule event-url="https://cfp.matrix.org/matrixconf2024/" locale="en" format="grid"
-            style="--pretalx-clr-primary: #000000"></pretalx-schedule>
-        <noscript>
-            <div class="pretalx-widget">
-                <div class="pretalx-widget-info-message">
-                    JavaScript is disabled in your browser. To access our schedule without JavaScript,
-                    please <a target="_blank" href="https://cfp.matrix.org/matrixconf2024/schedule/">click
-                        here</a>.
-                </div>
-            </div>
-        </noscript>
-        -->
+                <h2>Schedule</h2>
+                <p>
+                    It's still too early to unveil our schedule for 2025! We will publish a Call for Proposals in the
+                    coming days.
+                </p>
+                <p>
+                    Stay tuned by following us on <a href="https://matrix.org/blog/">the Matrix Blog</a>, our <a
+                        href="https://uk.linkedin.com/company/matrix-org">LinkedIn account</a> or by <a
+                        href="https://mastodon.matrix.org/@matrix">following us on
+                        the fediverse</a>!
+                </p>
```

Comment:
I wish the block containing this text was somehow centered; it looks off to me
![image](https://github.com/user-attachments/assets/ea8e16ba-73da-434b-a11f-c7c6e5f0c4cf)